src/components/SingleList.tsx

```diff
@@ -1,13 +1,19 @@
-import './SingleList.css';
-import * as api from '../api/firebase';
+import "./SingleList.css";
+import * as api from "../api/firebase";
+import { useNavigate } from "react-router-dom";
 
-interface Props extends Pick<api.List, 'name' | 'path'> {
+interface Props extends Pick<api.List, "name" | "path"> {
 	setListPath: (path: string) => void;
 }
 
 export function SingleList({ name, path, setListPath }: Props) {
+	const navigate = useNavigate();
+
 	function handleClick() {
 		setListPath(path);
+		setTimeout(() => {
```

Comment:
praise 🎉: i know it seems like there are some other ideas being discussed about how to hadnle the little life switch over delay, so if that changes 👍🏽 BUT i really want to acknowledge how smart and cool this use of the `setTimeout` is! Was just coming back to see any new chagnes and it is so smooth now!